From 14e610532921448bb56332de502819b3607a18bf Mon Sep 17 00:00:00 2001
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 4 Jan 2022 20:44:50 +0000
Subject: [PATCH 1/5] gnu: asdf-transform-schemas: Enable tests

* gnu/packages/astronomy.scm: (asdf-transform-schemas): Enable tests.
  [native-inputs]: Add python-pytest
---
 gnu/packages/astronomy.scm | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

diff --git a/gnu/packages/astronomy.scm b/gnu/packages/astronomy.scm
index 77f8cfa024..e7c1d6fbf6 100644
--- a/gnu/packages/astronomy.scm
+++ b/gnu/packages/astronomy.scm
@@ -1281,11 +1281,16 @@ (define-public python-asdf-transform-schemas
         (base32 "1gmzd81hw4ppsvzrc91wcbjpcw9hhv9gavllv7nyi7qjb54c837g"))))
     (build-system python-build-system)
     (arguments
-     (list
-      ;; FIXME: (Sharlatan-20211229T160902+0000): tests run via tox
-      #:tests? #f))
+     `(#:phases
+       (modify-phases %standard-phases
+         (replace 'check
+           (lambda* (#:key inputs outputs tests? #:allow-other-keys)
+             (when tests?
+               (add-installed-pythonpath inputs outputs)
+               (invoke "python" "-m" "pytest")))))))
     (native-inputs
-     (list python-semantic-version
+     (list python-pytest
+           python-semantic-version
            python-setuptools-scm))
     (propagated-inputs
      (list python-asdf))
@@ -1296,8 +1301,6 @@ (define-public python-asdf-transform-schemas
 should not need to install this directly; instead, install an implementation
 package such as asdf-astropy, which includes asdf-transform-schemas as a
 dependency.")
-    ;; Copyright (C) 2021 Association of Universities for Research in Astronomy (AURA)
-    ;; https://github.com/asdf-format/asdf-transform-schemas/blob/master/LICENSE
     (license license:bsd-3)))
 
 (define-public python-asdf-coordinates-schemas
-- 
2.34.0

